schemas/DatabaseField.schema

```diff
@@ -31,14 +32,14 @@
 
     // Validate calculated field
     if (data.expression) {
-      return allowed(['domain', 'category', 'control', 'comment']);
+      return allowed(['domain', 'display', 'category', 'control', 'comment']);
```

Comment:
:thinking: shouldn't it be `allow(..)`?

Reply:
We have `allowed(allow)` but you are right, it would be better to have `allow(allowed)`